### PR TITLE
HTS-761 Implemented service location and documentation controller

### PR DESCRIPTION
**Pull request by Jackie1988**

### CODE REVIEW

File: app/uk/gov/hmrc/helptosaveapi/connectors/ServiceLocatorConnector.scala [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    val metadata: Option[Map[String, String]] = Some(Map("third-party-api" -> "true"))
+    Registration(appName, appUrl, metadata)
+  }
+
+  def register(): Future[Boolean] = {
+    http.post(s"$serviceUrl/registration", registration).map{
+      _.status === 204
+    } recover {
+      case e: Throwable ⇒
+        false
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/helptosaveapi/connectors/ServiceLocatorConnector.scala]: you can log a warning here which tell whats the error was

File: project/MicroService.scala [hunk 2]
```diff
@@ -94,6 +95,8 @@ trait MicroService {
     .settings(defaultSettings(): _*)
     .settings(scalariformSettings: _*)
     .settings(wartRemoverSettings)
+    .settings(unmanagedResourceDirectories in Compile += baseDirectory.value / "resources")
+    .settings(unmanagedResourceDirectories in Test += baseDirectory.value / "resources")
```
Comment (sureshhmrc) [inline project/MicroService.scala]: now you know how to read from top level `resources` folder , instead of copying the raml into tests :)

Comment (Jackie1988) [reply project/MicroService.scala]: Yeah its good to know :)

File: app/uk/gov/hmrc/helptosaveapi/connectors/ServiceLocatorConnector.scala (hunk 1 above)
Comment (Jackie1988) [reply app/uk/gov/hmrc/helptosaveapi/connectors/ServiceLocatorConnector.scala]: Yeah sure

**Pull request merged**